public/500.html

```diff
@@ -6,6 +6,6 @@
 <body>
   <h1>Lobsters HTTP 500 status: server error</h1>
   <p>The page you requested cannot be displayed.</p>
-  <p>The site operators have been notified of this error.  You're welcome to and encouraged to also report it in <a href="/chat">chat</a>.</p>
+  <p>The site operators have been notified of this error.  You're welcome to and encouraged to also report it in the <a href="irc://irc.freenode.net/lobsters">IRC channel</a> (also accessible via <a href="https://webchat.freenode.net/#lobsters">webchat<a>).</p>
```

Comment:
I just noticed the closing anchor on the webchat a tag is not a closing tag, as it is missing a forward-slash (/):

<a href="">anchor text</a>